projects/plutus/src/fault_monitor.c

```diff
@@ -3,23 +3,32 @@
 #include "log.h"
 #include "plutus_event.h"
 
+static int s_i = 0;
```

Comment:
Comment to explain what this is for please. Looks unused?

Reply:
Debug. I'll remove it.